CODING_STYLE.md

```diff
@@ -0,0 +1,39 @@
+# Coding Style
+
+## General
+
+When contributing to the project, and if otherwise not mentioned in this document, our coding conventions
+follow the Microsoft [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions)
+and standard [Naming Guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines).
+
+## Class Members
+
+Members and types should always have the lowest possible visibility.
+
+Ordering of class members should be the following:
```

Comment:
Using [live preview](https://markdownlivepreview.com/), this section does not work and get's collapsed to single paragraph. There should be empty line before the first item.